src/main/java/com/cascadebot/cascadebot/commands/core/ServerInfoCommand.java

```diff
@@ -5,28 +5,43 @@
 
 package com.cascadebot.cascadebot.commands.core;
 
+import com.cascadebot.cascadebot.CascadeBot;
 import com.cascadebot.cascadebot.commandmeta.CommandContext;
 import com.cascadebot.cascadebot.commandmeta.CommandType;
 import com.cascadebot.cascadebot.commandmeta.ICommand;
-import com.cascadebot.cascadebot.messaging.MessagingObjects;
+import com.cascadebot.cascadebot.utils.DiscordUtils;
 import com.cascadebot.cascadebot.utils.FormatUtils;
+import net.dv8tion.jda.bot.sharding.ShardManager;
 import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Guild;
 import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.entities.Message;
 import java.util.Set;
 
 public class ServerInfoCommand implements ICommand {
     @Override
     public void onCommand(Member sender, CommandContext context) {
+        Guild guildForInfo = context.getGuild();
+        Message userMessage = context.getMessage();
+
+        if(context.getArgs().length > 0) {
+            guildForInfo = CascadeBot.INS.getShardManager().getGuildById(context.getMessage(0));
```

Comment:
use getArg instead, this gets multiple arguments which can cause errors

Reply:
How would I go about doing that? If I change getArgs() to getArg() it pops up with an error.

Reply:
Not getArgs, change getMessage(0) to getArg(0)